src/stores/optimade.ts

```diff
@@ -17,7 +17,7 @@ type StructuresByProviders = Array<[Types.StructuresResponse[], Types.Provider]>
 
 const getStructuresAll = debounce(
 	(providers: string[], filter: string, page: number, limit: number, batch: boolean) => {
-		return optimade.getStructuresAll(providers, filter, page, limit, batch) || [];
+		return optimade.getStructuresAll({ providers, filter, page, limit, batch }) || [];
```

Comment:
@Valexr only this is the fix for the new client, right? The other edits are fully about something else.

Reply:
Full PR for new client.